src/styles/SearchResultsStyle.css

```diff
@@ -0,0 +1,19 @@
+.card-image{
+    display: flex;
+    justify-items: auto;
+    width: 30%;
+    height: 50%;
+    margin: auto;
+    padding: 5px;
```

Comment:
Im not sure what you're trying to do here. The height property isnt doing anything, and `justify-items` does not have an auto option. Also, since there is margin applied on all sides the padding is pretty redundant. It does functionally work, but could be tidied up a bit